packages/core/src/progress/LinearProgress/LinearProgress.tsx

```diff
@@ -56,8 +62,11 @@ export const LinearProgress = forwardRef<HTMLDivElement, LinearProgressProps>(
       window: targetWindow,
     });
 
-    const progress = ((value - min) / (max - min)) * 100;
-    const buffer = ((bufferValue - min) / (max - min)) * 100;
+    const isIndeterminate = variant === "indeterminate";
+    const progress = isIndeterminate ? 66 : ((value - min) / (max - min)) * 100;
```

Comment:
66 is a magic number, I have to read through the code to understand what it means, assigning it to a variable would make the value clearer. 
 
 How did we arrive at 66 ?

Reply:
This was a value agreed by design, I'll move it into a variable